script.js

```diff
@@ -126,3 +126,26 @@ shuffleButton.addEventListener('click', () => {
     );
   }
 });
+
+document.getElementById('start-quiz').addEventListener('click', () => {
+  const player1Name = document.getElementById('player1-name').value;
+  const player2Name = document.getElementById('player2-name').value;
+  if (player1Name && player2Name) {
+    document.getElementById('player1-display').textContent = player1Name;
+    document.getElementById('player2-display').textContent = player2Name;
+    document.getElementById('players-container').style.display = 'block';
+  } else {
+    alert('Please enter both player names!');
+  }
+});
+
+function updateScore(player, action) {
```

Comment:
A cool idea to use the same handler function for both the Correct and Wrong options, and just tell the function which one to handler using the `action` variable. ⭐